conduit/src/Data/Conduit/Combinators.hs

```diff
@@ -304,6 +301,21 @@ import           Data.Sequences (LazySequence (..))
 --   so there's no way for the conduit to guarantee a certain amount
 --   of demand from the upstream.
 
+-- $naming_scheme
+--
+-- [@Functions ending with C@]: This postfix indicates a function returns a
+--   'ConduitM'
+--
+-- [@Functions starting with o@]: This comes from \"mono-traversable\". The o
+--   choice is a bit strange, but it makes sense when you realize that the
```

Comment:
This only appears to functions which are monomorphic, like `omap`. Also worth clarifying.